plugins/osdn/common.go

```diff
@@ -29,6 +29,7 @@ type OvsController struct {
 	Registry        *Registry
 	localIP         string
 	localSubnet     *api.Subnet
+	clusterNet      *net.IPNet
```

Comment:
Um, that comment applies to the old code, not the current code. But it's ok in the current code too.

But it's weird that the field is only set for masters, not for nodes. And that in some places, you've plumbed clusterNetworkCIDR through (eg, PluginStartNode()), but in other places, it uses the field.

Perhaps we should just make Registry.serviceNetwork and Registry.clusterNetwork public so we can refer to them everywhere? Although then there's the whole set of issues that went with making sure those get set right (https://github.com/openshift/origin/pull/6684) which might apply to oc.clusterNet too?